ote_cli/ote_cli/tools/ote.py

```diff
@@ -59,6 +60,7 @@ def main():
       - find
       - train
       - optimize
+      - explain
     """
 
```

Comment:
Please import `from .explain import main as ote_explain` at the top

Reply:
Sorry, added the function and import